src/pathways/tags/tags.controller.ts

```
@@ -0,0 +1,323 @@
+  async list(
+    @Body() listTagDto: ListTagDto,
+    @Headers('tenantid') tenantId: string,
+    @Res() response: Response
+  ): Promise<Response> {
+    if (!tenantId || !isUUID(tenantId)) {
+      throw new BadRequestException(API_RESPONSES.TENANTID_VALIDATION);
+    }
+    return this.tagsService.list(listTagDto, response);
+  }
+
+  @Post('fetch/:id')
```

ESLint

[error] Replace `'fetch/:id'` with `"fetch/:id"`

(prettier/prettier)